lib/datadog/ci/contrib/minitest/runnable.rb

```diff
@@ -0,0 +1,46 @@
+require_relative "suite"
+
+module Datadog
+  module CI
+    module Contrib
+      module Minitest
+        module Runnable
+          def self.included(base)
+            base.singleton_class.prepend(ClassMethods)
+          end
+
+          module ClassMethods
+            def run(*)
+              return super unless datadog_configuration[:enabled]
+              return super if parallel?
+
+              method = runnable_methods.first
+              return super if method.nil?
+
+              test_suite_name = Suite.name(self, method)
+
+              test_suite = Datadog::CI.start_test_suite(test_suite_name)
+              test_suite.passed! # will be overridden if any test fails
```

Comment:
Does it mean a test suite can never have 'skipped' status? Not sure what the intended behaviour is, mentioning it just in case

Reply:
Yes, I haven't implemented `SKIP` for test suites in any of the instrumentations right now. Is it heavily used by other libraries (if we don't include ITR)?

Reply:
In Java the suite status is SKIP if it contains 0 test cases, or if every test case in it is skipped. Not sure about other languages

Reply:
thank you! I will add a note for the future improvements